pkg/execute/filter.go

```diff
@@ -0,0 +1,139 @@
+package execute
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"text/tabwriter"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/kubeshop/botkube/pkg/bot/interactive"
+	"github.com/kubeshop/botkube/pkg/config"
+	"github.com/kubeshop/botkube/pkg/execute/command"
+	"github.com/kubeshop/botkube/pkg/filterengine"
+)
+
+const (
+	filterNameMissing = "You forgot to pass filter name. Please pass one of the following valid filters:\n\n%s"
+	filterEnabled     = "I have enabled '%s' filter on '%s' cluster."
+	filterDisabled    = "Done. I won't run '%s' filter on '%s' cluster."
+)
+
+var (
+	filterResourcesNames = []string{"filter", "filters"}
+)
+
+// TODO: Refactor as a part of https://github.com/kubeshop/botkube/issues/657
```

Comment:
Keeping this here, as I don't think current changes are providing the desired fix